app/assets/javascripts/components/uploads/upload.jsx

```diff
@@ -39,22 +23,22 @@ const Upload = ({ upload, linkUsername }) => {
     uploader = upload.uploader;
   }
 
+  let usage = '';
+  if (upload.usage_count) {
+      usage = `${upload.usage_count} usage${upload.usage_count !== 1 ? 's' : ''}`;
```

Comment:
This should be done with an i18n message to which you pass the count. There's already a message, `uploads.usages`, which I'm not sure where it's used. You might be able to repurpose that and add pluralization support, or you might need to add a new message key.